libs/portals/admin/ids-admin/src/components/Applications/Applications.css.ts

```diff
@@ -0,0 +1,20 @@
+import { globalStyle, style } from '@vanilla-extract/css'
+import { theme } from '@island.is/island-ui/theme'
+
+export const relative = style({
+  position: 'relative',
+})
+
+globalStyle(`${relative} a`, {
```

Comment:
Does this have to be global?

Reply:
It is only "global" inside this "relative" class, so it will apply to all "a" tags inside of it.